test/Interop/Cxx/templates/Inputs/linkage.h

```diff
@@ -0,0 +1,21 @@
+#ifndef TEST_INTEROP_CXX_TEMPLATES_INPUTS_LINKAGE_H
```

Comment:
I'd suggest to rename this header to "linkage-of-swift-symbols-for-imported-types" (and similarly the tests that use it).

Reply:
Done.